packages/react-magma-dom/src/components/Table/TablePagination.tsx

```diff
@@ -108,6 +108,7 @@ const StyledContainer = styled.div<{
   display: flex;
   justify-content: flex-end;
   padding: ${props => props.theme.spaceScale.spacing02};
+  border-radius: 0 0 ${props => props.theme.borderRadius} ${props => props.theme.borderRadius};
```

Comment:
Do we need to incorporate a similar "hasSquareCorners" property that toggles between this and the previous behavior? I am a bit wary of bluntly making a style change that can't be reverted.

Reply:
I spoke to our designer @orion-cengage and we agreed that we _should_ add a `hasSquareCorners` to match the prop for tables